### PR TITLE
Remove all use of 2nd person from ugprade doc

### PR DESCRIPTION
**Pull request by sirosen**
This was an issue with the first version of this doc, but we decided to push it out without fixing it yet.

Using the second person in docs is a bad habit of mine. I just took a few minutes to go through and do the necessary rewrites.

### CODE REVIEW

File: docs/upgrading.rst [hunk 1]
```diff
@@ -34,9 +34,9 @@ parameter, you must be explicit and rewrite like so:
     def foo(args):
         return some_function(user_query=args.get("q"))
 
-This also means that another usage from 5.x is not supported. If you had code
-with multiple locations in a single `use_args`, `use_kwargs`, or `parse` call,
-you must write it in multiple separate `use_args` or `use_kwargs` invocations,
+This also means that another usage from 5.x is not supported. Dode with
```
Comment (lafrech) [inline docs/upgrading.rst]: Code

File: docs/upgrading.rst [hunk 2]
```diff
@@ -118,15 +120,14 @@ the schema. In order to achieve this, webargs now passes the full data from
...
-types which can return one or multiple values as you prefer. To handle
-`marshmallow.fields.List` and `webargs.fields.DelimitedList` fields correctly,
-passing list data, webargs must combine schema information with the raw request
-data. This is done in the
+types which can return one or multiple values. To handle `marshmallow.fields.List`
+and `webargs.fields.DelimitedList` fields correctly, passing list data, webargs
+must combine schema information with the raw request data. This is done in the
 :class:`MultiDictProxy <webargs.multidictproxy.MultiDictProxy>` type, which
-will often be passed to your schemas.
+will often be passed to schemas.
```
Comment (lafrech) [inline docs/upgrading.rst]: Should the whole paragraph be moved to line 180, with "Finally, this change passes a proxy object where schemas once saw a dict."?

Looks like this chapter interleaves unknown and multidict issues. Or maybe I read too fast.

**Review by lafrech — APPROVED** [review]
Just two comments.

Looks good otherwise.

👍

**Comment by sirosen** [comment]
I've made a couple of changes for review and pushed in a new commit to make it easy to see.

I agree with moving that paragraph about MultiDictProxy down to go with the doc on how to adapt usage. Looking at that section, it's all about "making changes to data passed to schemas", so it holds together logically. We could split it up if you want, but I think it's okay?

**Pull request merged**